Makefile

```
@@ -27,9 +27,8 @@ DOCKER_WORKDIR?=/go/src/github.com/argoproj/argo-cd
 
 ARGOCD_PROCFILE?=Procfile
 
-# Strict mode has been disabled in latest versions of mkdocs-material.
-# Thus pointing to the older image of mkdocs-material matching the version used by argo-cd.
-MKDOCS_DOCKER_IMAGE?=squidfunk/mkdocs-material:4.1.1
+# pointing to python 3.8 as the oldest version of python supported and compatible with argocd-docs mkdocs/mkdocs-material
+MKDOCS_DOCKER_IMAGE?=python:3.8-alpine
```

Should we match the python version to [what's used in `.readthedocs`](https://github.com/argoproj/argo-cd/blob/2ac96a55badf673cf16acc4a959ac7b3e325e1cf/.readthedocs.yml#L11)? 
```suggestion
MKDOCS_DOCKER_IMAGE?=python:3.7-alpine
```

---

See my comment
>The Python Software Foundation (PSF) will no longer support Python 3.7 on June 27th, 2023

We should try bump to 3.8 for the [.readthedocs.yaml](https://github.com/argoproj/argo-cd/blob/master/.readthedocs.yml) but that would need to be done in a different PR and do testing with .readthedocs build system

---

let me check if the image `3.7-alpine` is availble

---

Correction, I just saw the comment, should we update what's in the .readthedocs to 3.8 then?

---

its looks like is available, I will update to 3.7, and then see when can we move everything to a newer version of python 3.8+

---

3.8 in .readthedocs should be a different issue/pr combination